Mathlib/LinearAlgebra/Finsupp.lean

```diff
@@ -636,9 +636,14 @@ theorem linearCombination_zero : linearCombination R (0 : α → M) = 0 :=
 
 variable {α M}
 
+theorem linearCombination_comp_linear (f : M →ₗ[R] M') :
```

Comment:
What about `linearCombination_linear_comp` as `f` is on the left?

Reply:
Sure, renamed